doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -223,17 +251,24 @@ The ECC algorithm support is dependent on one or more Kconfig options enabling c
 ECC driver configurations
 =========================
 
-You can use the following Kconfig options for fine-grained control over which drivers provide ECC support:
+You can use the following table to check the ECC algorithm support of each driver:
+
++-----------------------+---------------------------+----------------------------+
+| ECC algorithm         | nrf_cc3xx driver support  | nrf_oberon driver support  |
++=======================+===========================+============================+
+| ECDH                  | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| ECDSA                 | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| ECDSA (deterministic) | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+
+
+The option :kconfig:option:`PSA_USE_CC3XX_SIGNATURE_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for the ECDSA and ECDSA deterministic algorithms.
```

Comment:
The term "SIGNATURE_DRIVER" is a bit vague and would be applicable to RSA algorithms as well. Intentional?